src/app/pages/news-article/news-article.page.html

```diff
@@ -43,6 +43,27 @@
       <div [innerHTML]="article.content" ></div>
       <!--{{article.textBody}}-->
     </ion-card-content>
+
+    <ng-container *ngIf="checkIfAnyRelatedArticles()">
+      <h2 style="padding-left: 55px; padding-bottom: 5px;">Related Articles</h2>
```

Comment:
Can you move all styling to the .scss file? 

I'm sure I wasn't following that structure when this was being built, but it is actually best to not do inline styling and use classes/ids instead

Reply:
Yes, of course